Source/ParameterEncoding.swift

```diff
@@ -85,13 +85,17 @@ public struct URLEncoding: ParameterEncoding {
         case brackets
         /// No brackets are appended. The key is encoded as is.
         case noBrackets
+        /// Index in square brackets ("key[index]")
```

Comment:
This should be phrased similarly to the other cases. Something like "Brackets containing the item index appended. This matches the jQuery and Node.js behavior."